strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/MavenArtifactDeployer.java

```diff
@@ -64,29 +64,32 @@ public void generateAndDeployArtifact(Artifact artifact,
                    IOException,
                    ArtifactOperationException, ArtifactTransportException
     {
-        generatePom(artifact, packaging);
-        createArchive(artifact);
+        synchronized (ArtifactUtils.convertArtifactToPath(artifact))
```

Comment:
@carlspring  @fuss86 I don't know who is the author of this `synchronized` but we shouldn't synchronize like this.

also, this artifact generator should already be thread safe in `master` and this sync probably needs to be removed

Reply:
I agree.

Reply:
reverted